src/olympia/lib/settings_base.py

```diff
@@ -1362,7 +1372,12 @@ def get_db_config(environ_var, atomic_requests=True):
         },
         'z': {
             'handlers': ['mozlog'],
-            'level': logging.DEBUG,
+            'level': logging.INFO,
+            'propagate': False
+        },
+        'z.addons': {
+            'handlers': ['mozlog'],
+            'level': logging.INFO,
             'propagate': False
```

Comment:
I suppose this is the default config now (inherited from `root`) so we could probably remove this entry.

Reply:
also yes